packages/wdio-browserstack-service/src/types.ts

```diff
@@ -1,7 +1,7 @@
 import type { Capabilities, Options, Frameworks } from '@wdio/types'
 import type { Options as BSOptions } from 'browserstack-local'
 
-export type MultiRemoteAction = (sessionId: string, browserName?: string) => Promise<any>;
+export type MultiRemoteAction = (sessionId: string, browserName?: string) => Promise<unknown>;
```

Comment:
I think this function doesn't return a value, if so this can be Promise<void>

Reply:
I have not validated which type this function returns, so to be safe I set it to `unknown`.